qiskit/test/mock/backends/guadalupe/fake_guadalupe.py

```diff
@@ -0,0 +1,47 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Fake Guadalupe device (5 qubit).
+"""
+
+import os
+import json
+
+from qiskit.providers.models import QasmBackendConfiguration, BackendProperties
+from qiskit.test.mock.fake_backend import FakeBackend
+
+
+class FakeGuadalupe(FakeBackend):
```

Comment:
This is a base class that should be used directly here, instead you should have used `qiskit.test.mock.fake_pulse_backend.FakePulseBackend` and then just defined the class attributes for the filenames.

Reply:
Good catch! Thanks. I'll submit a follow up.

Reply:
No worries, I'veggot it covered as partof a larger refractor in #6286